tests/all.rs

```diff
@@ -34,6 +35,7 @@ pub mod rational_dimensions;
 
```

Comment:
I think this needs a 
```
#[cfg(any(feature = "si", feature = "num-traits-libm"))]
```
as well, since it`s calling `sqrt`.

Reply:
I wish we could tell cargo that the `rational-dimensions` feature depends on one of {`std`, `num-traits-libm`}. I don't think it can be used at all (not just in the tests), so I'd rather make it an early compile-time error instead of just hiding the test.

Reply:
Why is that? `cargo test --no-default-features --features f32,f64,rational-dimensions` seems to run fine for me, is there something I am missing?
Of course, the usefulness of the feature is certainly debatable without even being able to call `sqrt`.

Reply:
That's what I mean: I can't think of a reason a user would deliberately enable `rational-dimensions` without ability to use the likes of `sqrt`, and thus we'd be doing them a favor by reminding them eagerly.

Reply:
I see! We could add a 
```rust
#[cfg(all(not(any(feature = "si", feature="num-traits-libm")), feature="rational-dimensions"))]
compile_error!("Rational dimensions are of no use without either `num-traits-libm` or `std`, consider compiling without the `rational-dimensions` feature or enabling one of `num-traits-libm` or `std`.");
```

(might be a few misaligned parantheses there, it's late :) )

Reply:
oops, just realized you did the exact same thing below. nevermind that